packages/table/src/table.tsx

```diff
@@ -730,6 +730,13 @@ export class Table extends AbstractComponent<ITableProps, ITableState> {
             newSelectedRegions,
         );
 
+        if (
+            !CoreUtils.arraysEqual(newColumnWidths, this.state.columnWidths) ||
+            !CoreUtils.arraysEqual(newRowHeights, this.state.rowHeights)
+        ) {
+            this.didUpdateColumnOrRowSizes = true;
```

Comment:
I see that in other places we call `this.invalidateGrid()` when we set this to `true`. It looks like that's not strictly necessary since it's called 4 lines later, but maybe add a comment

Reply:
I could just move up the `invalidateGrid()` call to just below this block? Not sure what I would be trying to explain in a comment